app.js

```diff
@@ -13,58 +13,77 @@ const catchphraseButton = document.getElementById('catchphrase-button');
 // set state for how many times the user changes the head, middle, and bottom
 // set state for all of the character's catchphrases
 
+let headCount = 0;
+let middleCount = 0;
+let bottomCount = 0;
+
+const catchPhrases = [];
+
 headDropdown.addEventListener('change', () => {
     // get the value of the head dropdown
-
+    const headSelect = headDropdown.value; 
     // increment the head change count state
-    
+    headCount++;
     // update the dom for the head (use style.backgroundImage on the bottomEl div instead of trying to set the .src -- it's NOT an img tag!)
-
+    headEl.style.backgroundImage = `url(./assets/${headSelect}-head.png)`;
     // update the stats to show the new count (call displayStats() to do this work)
+    displayStats();
+
 });
 
 
 middleDropdown.addEventListener('change', () => {
     // get the value of the middle dropdown
-
+    const middleSelect = middleDropdown.value;
     // increment the middle change count state
-    
+    middleCount++;
     // update the dom for the middle (NOTE: use style.backgroundImage on the middleEl div instead of trying to set the .src -- it's NOT an img tag!)
-
+    middleEl.style.backgroundImage = `url(./assets/${middleSelect}-middle.png)`;
     // update the stats to show the new count (call displayStats() to do this work)
+    displayStats();
 });
 
 
 bottomDropdown.addEventListener('change', () => {
     // get the value of the bottom dropdown
-
+    const bottomSelect = bottomDropdown.value;
     // increment the bottom change count state
-    
+    bottomCount++;
     // update the dom for the bottom (NOTE use style.backgroundImage on the bottomEl div instead of trying to set the .src -- it's NOT an img tag!)
-
+    bottomEl.style.backgroundImage = `url(./assets/${bottomSelect}-pants.png)`;
     // update the stats to show the new count (call displayStats() to do this work)
+    displayStats();
+
 });
 
 catchphraseButton.addEventListener('click', () => {
     // get the value of the catchphrase input
-    
+    const getcatchPhrases = catchphraseInput.value;
     // push the new catchphrase to the catchphrase array in state
-
+    catchPhrases.push(getcatchPhrases);
     // clear out the form input's value so it's empty to the user
-   
-    // update the dom to show the new catchphrases (refactor to/call displayCatchphrases to do this work)
 
```

Comment:
here if you add `catchphraseInput.value = '';` it will clear out the input field after the user clicks

Reply:
I got it! Ty